trigger.js

```diff
@@ -14,6 +14,8 @@ class TriggerHappy {
         Hooks.on('hoverToken', this._onHoverToken.bind(this));
         Hooks.on('updateJournalEntry', this._onUpdateJournal.bind(this));
         Hooks.on('deleteJournalEntry', this._onUpdateJournal.bind(this));
+        Hooks.on("preUpdateToken", this._onEndUpdate.bind(this));

```

Comment:
Yeah I played around with onUpdate and onPreUpdate hence the onEndUpdate name, But you're right can change the name now. It was pretty funny when it was hooking of onUpdate instead since the downstream macro was creating and deleting tokens swtiching scenes sorts of things that made for a very confusing experience when it was being done my multiple clients.